tests/main/presenters/test_search_summary.py

```diff
@@ -266,6 +266,38 @@ def test_subcategory_filters_are_available(self):
         summary_markup = search_summary.markup()
         assert "category <em>Analytics</em>" in summary_markup
 
+    def test_each_filter(self):
+        """
+        Test each filter individually adds a single text string to the summary text
+        """
+        for lot in self._g9_lots_by_slug:
+            self.request_args['lot'] = lot
+            for filter_group in g9_filter_groups:
+                for f in filter_group['filters']:
+                    request_args = self.request_args.copy()
+                    request_args[f['name']] = f['value']
+                    search_summary = SearchSummary('9', request_args, g9_filter_groups, self._g9_lots_by_slug)
+                    summary_markup = search_summary.markup()
+                    assert summary_markup.count("<em>") == 3  # the keyword, the lot, and one filter
```

Comment:
Heh heh heh.

I see what this is doing... but it feels a bit weird to rely test it this way. Doesn't feel like it would be completely insane for us to decide one/more of these shouldn't be bold, and we'd break the test without it functionally breaking. If we're validating that the correct labels are displayed somewhere else, maybe this would be OK after adding some extra comments.